addon/utils/lookup-classes.js

```diff
@@ -0,0 +1,5 @@
+import data from '../css-classes-json';
```

Comment:
I'll add that to the misc issues to address (you pointed out a thing that probably should be configurable in some way).